src/dmx/Utilities.cpp

```diff
@@ -17,20 +17,20 @@ ColorBuffer::ColorBuffer()
 
 void ColorBuffer::setChannelValue(size_t channel, uint8_t value) {
 	channel = std::min(channel, _data.size() - 1);
```

Comment:
These constraints on the incoming channel make sure values aren't out of range. That way, we know directly accessing the data is safe (provided we have a non-zero-length data array).

Using `at(index)` will throw an exception if things are out of range. This adds a bit of performance overhead to the function that we don't really need, since we are already guaranteeing values are within range.